__tests__/any-of-labels.spec.ts

```diff
@@ -0,0 +1,106 @@
+import {Issue} from '../src/classes/issue';
+import {IssuesProcessor} from '../src/classes/issues-processor';
+import {IIssuesProcessorOptions} from '../src/interfaces/issues-processor-options';
+import {DefaultProcessorOptions} from './constants/default-processor-options';
+import {generateIssue} from './functions/generate-issue';
+
+describe('any-of-labels option', () => {
+  test('should do nothing when not set', async () => {
+    const sut = new IssuesProcessorBuilder()
+      .emptyAnyOfLabels()
+      .issues([{labels: [{name: 'some-label'}]}])
+      .build();
+
+    await sut.processIssues();
+
+    expect(sut.staleIssues).toHaveLength(1);
```

Comment:
I saw that the other tests are not using this method to check the length yet it's the most appropriate one ❤️

Reply:
Yeah I'm still not 100% satisfied with that one cause there is some implementation detail leaking out. It would probably be worth it to explore adding a custom matcher for this in the future, maybe? Like `expect(sut).toHaveProcessedIssues()`.